test/utils/envtest/environment.go

```diff
@@ -44,15 +44,18 @@ func New(projectRoot string) (*Environment, error) {
 		return nil, err
 	}
 	testBinPath := filepath.Join(projectRoot, "tmp", "test", "bin")
-	if err := os.Setenv("TEST_ASSET_KUBE_APISERVER", filepath.Join(testBinPath, "kube-apiserver")); err != nil {
-		return nil, err
-	}
-	if err := os.Setenv("TEST_ASSET_ETCD", filepath.Join(testBinPath, "etcd")); err != nil {
-		return nil, err
-	}
-	if err := os.Setenv("TEST_ASSET_KUBECTL", filepath.Join(testBinPath, "kubectl")); err != nil {
-		return nil, err
+	if _, err := os.Stat(testBinPath); err == nil {
```

Comment:
The case `os.Stat(testBinPath) != nil` is not handled

Reply:
that is by intention because if the testbindir is not set we rely on the kubebuilder defaults.
I added a comment to explain that behavior